version/version.go

```diff
@@ -7,5 +7,29 @@
 // Package version provides the version that the binary was built at.
 package version
 
+// LONG is a full version number for this build, of the form
+// "x.y.z-commithash", or "date.yyyymmdd" if no actual version was
```

Comment:
are we going to be appending a further `-gXXXXXX` to the `-commithash` for the corp repo's hash?

Reply:
As discussed, we're going for `1.2.3-tTAILSCALE_HASH-oOTHER_HASH`. The -o part is omitted in pure OSS builds.